Appraisals

```diff
@@ -1,5 +1,5 @@
 appraise "sass-3-4" do
-  gem "sass", "~> 3.4"
+  gem "sass", "~> 3.4.22"
```

Comment:
Would this level of specificity cause issues for projects that have versions `3.4.0..3.4.21`?